src/java/io/bazel/rulesscala/test_discovery/DiscoveredTestSuite.scala

```diff
@@ -0,0 +1,45 @@
+package io.bazel.rulesscala.test_discovery
+
+import org.junit.runner.RunWith
+import org.junit.runner.Runner
+import org.junit.runners.Suite
+import org.junit.runners.model.RunnerBuilder
+
+@RunWith(classOf[TextFileSuite])
+class DiscoveredTestSuite
```

Comment:
that's right. I'll add a comment though I'd rather try and change the name to make it more explicit. If it was called `TextFileSuiteEntryPoint` would that be clearer that this is just a "vessel"? an entry point? and that the interesting bit is in the `TextFileSuite`?